src/packages/block/block-type/components/input-block-type/input-block-type.element.ts

```diff
@@ -10,17 +11,33 @@ import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';
 import type { UmbPropertyDatasetContext } from '@umbraco-cms/backoffice/property';
 import { UMB_PROPERTY_DATASET_CONTEXT } from '@umbraco-cms/backoffice/property';
 import { UmbChangeEvent } from '@umbraco-cms/backoffice/event';
+import { UmbSorterController } from '@umbraco-cms/backoffice/sorter';
 
 @customElement('umb-input-block-type')
 export class UmbInputBlockTypeElement<
-	BlockType extends UmbBlockTypeBaseModel = UmbBlockTypeBaseModel,
+	BlockType extends UmbBlockTypeWithGroupKey = UmbBlockTypeWithGroupKey,
 > extends UmbLitElement {
+	#sorter = new UmbSorterController<BlockType, UmbBlockTypeCardElement>(this, {
+		getUniqueOfElement: (element) => element.contentElementTypeKey!,
+		getUniqueOfModel: (modelEntry) => modelEntry.contentElementTypeKey!,
+		identifier: 'block-editor-blocks-sorter',
+		itemSelector: '.cake',
```

Comment:
Ah, is that really the right name for this class.  You could just use `umb-block-type-card` no need for a class just to target it. :-)